src/chi-vue/src/components/data-table-tooltip/DataTableTooltip.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, Vue, Prop } from 'vue-property-decorator';
```

Comment:
As this component is Data Table specific and is not an independent module as toolbar or its sub-modules, could you move the file to `data-table` folder, please?

Reply:
sure